googlemaps/client.py

```diff
@@ -109,7 +116,12 @@ def __init__(self, key=None, client_id=None, client_secret=None,
         self.client_id = client_id
         self.client_secret = client_secret
         self.retry_timeout = timedelta(seconds=retry_timeout)
-
+        self.requests_kwargs = requests_kwargs or {}
+        self.requests_kwargs.update({
+            "headers": {"User-Agent": _USER_AGENT},
+            "timeout": self.timeout,
```

Comment:
I don't think we should forcibly override the timeout arg. Seems like a reasonable thing to want to change.